### DISCUSSION TITLE
Alguém usa scanners de erro de acessibilidade?

### DISCUSSION THREAD

**Discussion by daniellucindo** [frontendbr/forum#798] [Discussão]
E ai galera :3
Estou estudando aplicar Scanner de erros no meus projetos e encontrei esse:

Accessibilityjs
https://github.com/github/accessibilityjs

Alguém utiliza esse? se usa outro poderia compartilhar o conhecimento?

Abraços...

**Comment by lucasjs** [comment]
Boa tarde!

Bem legal esse que você passou, vou experimentar.

Conheço também esses:
- [wave.webaim.org](wave.webaim.org)
- [www.acessibilidade.gov.pt/accessmonitor](www.acessibilidade.gov.pt/accessmonitor)

**Comment by afonsopacifer** [comment]
Opa,
Aqui eu uso o [Accessibility Developer Tools](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb) do Chrome, no caso uso ele integrado na suite de testes unitários, ai sempre que rodo os testes ele da uma olhada na a11y.

PS: Eu acabo escrevendo alguns testes unitários voltados p a11y tbm, assim dá p garantir alguns pontos que o Accessibility Developer Tools não cobre (por ser algo especifico da aplicação), só de passar ele já ajuda d++++

:)

**Comment by williamcosta** [comment]
Tem esse vídeo que ajuda bastante a debuggar acessibilidade no Chrome:

https://www.youtube.com/watch?v=56zCnwj58e4

Para checar a acessibilidade eu já passei meus códigos por esse checker: https://achecker.ca/checker/index.php até hoje me atendeu bem.

Para teste de acessibilidade no vídeo eles citam esse cara [aXe-core](https://github.com/dequelabs/axe-core)
